### DISCUSSION TITLE
not familer with polyfill, but is it right`Symbol.asyncDispose` without esnext.symbol.async-dispose ?

### DISCUSSION THREAD

**Discussion by HomyeeKing** [babel/babel#15788] [General] [answered]
https://babeljs.io/repl#?browsers=&build=&builtIns=usage&corejs=3.21&spec=false&loose=false&code_lz=MoTwtgRg9gNgdAQwM4gHYGMAiBLJAHKJAUwCgg&debug=false&forceAllTransforms=false&modules=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=false&presets=env%2Creact%2Cstage-2&prettier=false&targets=&version=7.22.9&externalPlugins=&assumptions=%7B%7D

![image](https://github.com/babel/babel/assets/49113249/47a9a312-1672-43be-b3a9-69a2b8327013)

**Comment by JLHwung** [comment, marked as answer]
The `ayncDispose` is still a proposal, you will need to set `proposals: true` in `babel-plugin-polyfill-corejs3` option:

https://github.com/babel/babel-polyfills/tree/main/packages/babel-plugin-polyfill-corejs3#proposals

Unfortunately the REPL does not support doing so yet.
